pep-0484.txt

```diff
@@ -819,9 +819,11 @@ Example::
           e = [e]
       ...
 
-A type factored by ``Union[T1, T2, ...]`` responds ``True`` to
-``issubclass`` checks for ``T1`` and any of its subtypes, ``T2`` and
-any of its subtypes, and so on.
+A type factored by ``Union[T1, T2, ...]`` is compatible with
+``T1`` and any of its subtypes, ``T2`` and
+any of its subtypes, and so on. In other words, values of all types
+``T1``, ``T2``, etc., and their subtypes are acceptable for
+an argument annotated by ``Union[T1, T2, ...]``.
```

Comment:
We haven't defined "compatible" previously. Maybe you can avoid it and just use "supertype"? Or briefly define it here? (The IOW doesn't really do that IMO. :-)